@navikt/core/react/src/form/combobox/FilteredOptions/filteredOptionsContext.tsx

```diff
@@ -10,21 +10,18 @@ import React, {
 import { useClientLayoutEffect, usePrevious } from "../../../util/hooks";
 import { useInputContext } from "../Input/inputContext";
 import { useSelectedOptionsContext } from "../SelectedOptions/selectedOptionsContext";
+import { toComboboxOption } from "../combobox-utils";
 import { useCustomOptionsContext } from "../customOptionsContext";
-import { ComboboxProps } from "../types";
+import { ComboboxOption, ComboboxProps } from "../types";
 import filteredOptionsUtils from "./filtered-options-util";
 import useVirtualFocus, { VirtualFocusType } from "./useVirtualFocus";
 
 type FilteredOptionsProps = {
   children: any;
```

Comment:
Bør kunne bruke `React.ReactNode` som type her? (Har aversjon mot `any` 😝)
```suggestion
  children: React.ReactNode;
```